### PR TITLE
🐛  Do not parse hidden files like .gitkeep

### PR DESCRIPTION
**Pull request by thatInfrastructureGuy**
- 🐞 Fixes bug where it tries to parse .gitkeep are throws error `error loading task: filename does not encode UUID .gitkeep (wrong length)`
- 🧹 Use os.Readdir instead of deprecate ioutil.Readdir

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -113,8 +113,6 @@ github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
 github.com/mattn/go-isatty v0.0.12 h1:wuysRhFDzyxgEmMf5xjvJ2M9dZoWAXNNr5LSBS7uHXY=
 github.com/mattn/go-isatty v0.0.12/go.mod h1:cbi8OIDigv2wuxKPP5vlRcQ1OAZbq2CE4Kysco4FUpU=
-github.com/mvdan/xurls v1.1.0 h1:OpuDelGQ1R1ueQ6sSryzi6P+1RtBpfQHM8fJwlE45ww=
-github.com/mvdan/xurls v1.1.0/go.mod h1:tQlNn3BED8bE/15hnSL2HLkDeLWpNPAwtw7wkEq44oU=
```
Comment (naggie) [inline go.sum:117]: this seems like an unrelated change

**Comment by naggie** [comment]
Thanks @thatInfrastructureGuy  -- will merge soon

File: go.sum (hunk 1 above)
Comment (thatInfrastructureGuy) [reply go.sum]: Removed go.sum changes.
